### DISCUSSION TITLE
CONFIG FILE

### DISCUSSION THREAD

**Discussion by Mezzoura** [DiscordSRV/DiscordSRV#1738] [f. Questions] [answered]
when i download the plugin its just a file [DiscordSRV-Build-1.26.2.jar] and non folder of discordsrv or any file for config pls help

**Comment by J-295** [comment, marked as answer]
Follow the Inital Setup steps in the docs: https://docs.discordsrv.com/installation/initial-setup
Make sure to read everything carefully.
